general/press-and-publications.rst

```diff
@@ -0,0 +1,132 @@
+Press and Publications
+======================
+
+In this page, we aim to provide you links to presentations and popular
+blog posts about OpenWISP.
+
+Presentations:
+--------------
+
+`PyCon Italia 2017 <https://www.pycon.it/en/>`_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- video: https://www.youtube.com/watch?v=tm7Opg3QyZk
```

Comment:
you must link the word `video` and avoid showing URLs in the text because is very bad user experience, applies to all other lines